operations/observability/mixins/meta/rules/components/server/alerts.libsonnet

```
@@ -49,6 +49,105 @@
               description: 'Server cannot start workspace instances on workspace clusters.',
             },
           },
+          // Rollout alerts
+          {
+            alert: 'JSON RPOC API error rates',
```

```suggestion
            alert: 'JSONRPOCAPIerrorrates',
```

Alertmanager doesn't support spaces in alert names 😓

---

Ah, thx! Will fix.